### PR TITLE
docs: documentation for github home page

### PR DESCRIPTION
**Pull request by nkoppisetty**
Signed-off-by: nkoppisetty <nandita.iitkgp@gmail.com>

Home page document.

### CODE REVIEW

File: docs/quick-start.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+#activate virtual env
+poetry shell
+
+#install dependencies
+poetry install
+```
+
+## Using Jupyter Notebook:
+
+Start Jupyter Notebook, navigate to examples folder and open the quick start jupiter notebook. You can play with different availble options for each step.
```
Comment (vigith) [inline docs/quick-start.md:25]:
```suggestion
Start Jupyter Notebook, navigate to the examples folder and open the quick start Jupyter notebook. You can play with different available options for each step.
```

File: docs/quick-start.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+## Quick Start
+
```
Comment (vigith) [inline docs/quick-start.md:2]: content in quick start doesn't seem right. It should be the `curl` example. This is still very much ML specific. 

Are we planning to change this?

Comment (nkoppisetty) [reply docs/quick-start.md:2]: Yes, this is what I have right now. Will update with curl example in the next version.

**Review by vigith — APPROVED** [review]

**Review by ab93 — COMMENTED** [review]
Can you just make sure that the latest changes from https://github.com/numaproj/numalogic/pull/47 are rebased?

**Review by ab93 — APPROVED** [review]

**Pull request merged**